### DISCUSSION TITLE
ikev1 to ikev2 migration

### DISCUSSION THREAD

**Discussion by rodolpheAverty** [strongswan/strongswan#1515] [Q&A]

**Comment by rodolpheAverty** [comment]


**Reply by tobiasbrunner** [reply]
> I try to add a xfrm interface (not sure of what i am doing)

Then don't! You might want to read the [introduction](https://docs.strongswan.org/docs/5.9/howtos/introduction.html).

**Comment by tobiasbrunner** [comment]
> clientitineo{1}: 10.19.0.32/32 === 0.0.0.0/32

For some reason the peer reduces the proposed traffic selector `0.0.0.0/0` (as configured in `rightsubnet`) , which matches all IP addresses, to `0.0.0.0/32`, which does not match any IP addresses. Please check the configuration of the peer.
